src/pages-chibi/implementations/Anizium/main.ts

```diff
@@ -0,0 +1,151 @@
+import { PageInterface } from '../../pageInterface';
+
+export const Anizium: PageInterface = {
+  name: 'Anizium',
+  type: 'anime',
+  domain: 'https://anizium.co',
+  languages: ['Turkish'],
+  urls: {
+    match: ['*://anizium.co/*', '*://www.anizium.co/*', '*://m.anizium.co/*'],
```

Comment:
I think m.anizium.co does not exist. You can combine the other two to one by using '*://*.anizium.co/*'